src/chrome/content/rules/Boston.com.xml

```diff
@@ -1,97 +1,49 @@
 <!--
-	For rules causing false/broken MCB, see Boston.com-falsemixed.xml.
-
-	For other New York Times coverage, see NYTimes.xml.
-
+	Related:
+		BostonGlobe.com.xml
+		BostonGlobeMedia.com.xml
+		o0bc.com.xml
+		o0bg.com.xml
 
 	CDN buckets:
-
 		- nytbglobe.112.2o7.net
 		- s3.amazonaws.com/static.rondavu.com/C/Boston_PROD/
-
 		- wac.493C.edgecastcdn.net
-
-			- cinesport
-
+			- cinesport.boston.com
 		- cache.boston.com.global.prod.fastly.net
 		- bostoncom.http.internapcdn.net
-
 		- boston.nimblebuy.com
+			- deals.boston.com
 
-			- deals
-			- finds
-
-
-	Nonfunctional hosts in *boston.com:
-
-		- cinesport ¹
-		- realestate ²
-
-	¹ 404; mismatched, CN: gp1.wac.edgecastcdn.net
-	² Refused
-
-
-	Problematic hosts in *boston.com:
-
-		- deals ¹
-		- loveletters ²
-		- weather ³
-
-	¹ Works; mismatched, CN: finds.boston.com
-	² Mixed css
-	³ Refused
+	Mismatched:
+		cinesport.boston.com ( CN: gp1.wac.edgecastcdn.net )
 
+	Incomplete cer chain:
+		weather.boston.com
+		graphics.boston.com
 
-	Mixed content:
-
-		- css, on:
-
-			- loveletters from $self *
-			- loveletters from fonts.googleapis.com *
-
-		- Images, on:
-
-			- loveletters, www from $self *
-			- loveletters, www from cache.boston.com *
-
-		- favicon on www from p.o0bc.com
-		- Bug on loveletters, www from nytbglobe.112.2o7.net *
-
-	* Secured by us
+	MCB:
+		archive.boston.com
+		prototype.boston.com
 
+	Refused:
+		deals.boston.com
 -->
 <ruleset name="Boston.com (partial)">
```

Comment:
ae.boston.com
articles.boston.com
bostondirtdogs.boston.com
calendar.boston.com
circulars.boston.com
islandrun.boston.com
live.boston.com
people.boston.com
realestateads.boston.com
sponsored.boston.com
tickets.boston.com
yourcampus.boston.com

I think we can remove the partial flag then.